plutus-metatheory/src/Raw.lagda

```diff
@@ -85,6 +85,7 @@ decTermCon (bool b) (bool b') with b Data.Bool.≟ b'
 ... | yes p = true
 ... | no ¬p = false
 decTermCon unit unit = true
+-- TODO: char?
 decTermCon _ _ = false
```

Comment:
@jmchapman ^

Should we define boolean equality in terms of decidable equality? That wouldn't allow us to miss a case in the implementation.

Reply:
That sounds like a good plan, some of this stuff is a bit rough - perhaps make an issue for it?

Reply:
https://jira.iohk.io/browse/SCP-1432